src/modules/ekf2/EKF/airspeed_fusion.cpp

```diff
@@ -114,7 +114,11 @@ void Ekf::controlAirDataFusion(const imuSample &imu_delayed)
 			ECL_INFO("starting airspeed fusion");
 
 			// If starting wind state estimation, reset the wind states and covariances before fusing any data
-			if (!_control_status.flags.wind) {
+			// Also reset to the airspeed measurement if the current wind estimate has a low accuracy.
```

Comment:
Isn't this comment not entirely correct? How about
```suggestion
			// Also reset to the airspeed measurement if the current wind estimate has a lower accuracy than the expected airspeed observation accuracy. 
```

Reply:
I think I'll even rather indicate why we need this